src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Email addins released
+category: Release Notes
```

Comment:
I think Announcement would be better suited.  Release Notes is really reserved for Cake Releases.